docs/source/virtual_ecosystem/implementation/abiotic_implementation.md

```diff
@@ -113,109 +226,203 @@ Under steady-state, the balance equation for the leaves in each canopy layer is
 follows (after {cite:t}`maclean_microclimc_2021`):
 
 ```{math}
-    & R_{abs} - R_{em} - H - \lambda E \\
-    & = R_{abs} - \epsilon_{s} \sigma T_{L}^{4} - c_{P}g_{Ha}(T_{L} - T_{A})
+    & R_{abs} - R_{em} - H - Q_{LE} \\
+    & = R_{abs} - \epsilon_{s} \sigma T_{L}^{4} - \frac{\rho_a c_p}{r_a}(T_{L} - T_{A})
     - \lambda g_{v} \frac {e_{L} - e_{A}}{p_{A}} \\
     & = 0
 ```
 
-where $R_{abs}$ is absorbed radiation, $R_{em}$ emitted radiation, $H$
-the sensible heat flux, $\lambda E$ the latent heat flux, $\epsilon_{s}$ the
-emissivity of the leaf, $\sigma$ the Stefan-Boltzmann constant, $T_{L}$ the
-absolute temperature of the leaf, $T_{A}$ the absolute temperature of the air
-surrounding the leaf, $\lambda$ the latent heat of vapourisation of water,
-$e_{L}$ the effective vapour pressure of the leaf, $e_{A}$ the vapour
-pressure of air and $p_{A}$ atmospheric pressure. $g_{Ha}$ is the heat
-conductance between leaf and atmosphere, $g_{v}$ represents the conductance
-for vapour loss from the leaves as a function of the stomatal conductance $g_{c}$.
+where:
+
+$R_{abs}$:
```

Comment:
Same \text change applies here and a number of other places, it's basically a way to stop multiple letter subscripts from rendering in a funky way in LaTeX